system_test/helpers/aest_docker.erl

```diff
@@ -450,15 +440,18 @@ attempt_epoch_stop(#{container_id := ID, hostname := Name} = NodeState, Timeout)
     end,
     ok.
 
-maybe_continue_waiting(Id, infinity, StartTime) ->
-    timer:sleep(100),
-    wait_stopped(Id, infinity, StartTime);
-maybe_continue_waiting(Id, Timeout, StartTime) ->
-    case timer:now_diff(os:timestamp(), StartTime) > (1000 * Timeout) of
-        true -> timeout;
-        false ->
-            timer:sleep(200),
-            wait_stopped(Id, Timeout, StartTime)
+%% Sometime the stop command do not succeed...
```

Comment:
Personally I would remove this comment 🙂

Reply:
I thought one could wonder why we are retrying the stop command...

Reply:
The comment doesn't actually explain why though

Reply:
Because I don't really know why :/ Do we want to spend extra time to figure out why the command sometimes fail without returning any value?

Reply:
No, not really 🙂  Keep it then